config/initializers/assets.rb

```diff
@@ -1,7 +1,7 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+# Rails.application.config.assets.version = "1.0"
```

Comment:
We don't use the assets pipeline, and now that Spring (not the Java one) isn't included by default, this line causes errors on boot